.travis.yml

```diff
@@ -1,25 +1,52 @@
 language: python
 
+sudo: false
 
 env:
-    - CONDA="python=2.7"
-    - CONDA="python=3.4"
-    - CONDA="python=3.5"
-    - CONDA="python=3.6"
+  global:
+    - secure: "DdUE82IkNYo317595sy4H6z3GQoaVUTAAUDOOoJATa199ERxSQy8VlsMKELdRTrBEt76kSBJdjjJ7YSiOVSMt9qao9XWOWYjtfdX0KpbkmfiHMH6fmQnckNndMbPY8wa42fFusIWOxZL/jCay1Gq0GppjoHdSoaLGIk3hLkYE1U="
```

Comment:
Unfortunately I cannot publish the docs at TEOS-10/python-gsw because I don't have right there. This will publish them at the `gh-pages` of `ocefpaf/python-gsw` for now.